content/doc/book/installing/index.adoc

```diff
@@ -1,79 +1,334 @@
-This section is part of _Getting Started_.
-It provides instructions for *basic* Jenkins configuration on a number of platforms.
-It *DOES NOT* cover the full range of considerations or options for installing Jenkins.
-See link:/doc/book/appendix/advanced-installation/[Advanced Jenkins Installation]
-====
+Jenkins is typically run as a standalone application in its own process with the
+built-in
+link:https://stackoverflow.com/questions/7213541/what-is-java-servlet[Java
+servlet] container/application server
+(link:http://www.eclipse.org/jetty/[Jetty]).
 
-== Overview
+Jenkins can also be run as a servlet in different Java servlet containers such
+as link:http://tomcat.apache.org/[Apache Tomcat] or
+link:https://javaee.github.io/glassfish/[GlassFish]. However, instructions for
+setting up these types of installations are beyond the scope of this page.
 
+*Note:* Although this page focusses on local installations of Jenkins, this
+content can also be used to help set up Jenkins in production environments.
 
-== Pre-install
 
-=== System Requirements
+== Prerequisites
 
-[WARNING]
-====
-These are *starting points*.
-For a full discussion of factors see link:/doc/book/hardware-recommendations/[Discussion of hardware recommendations].
-====
+Minimum hardware requirements:
 
-Minimum Recommended Configuration:
+* 256 MB of RAM
+* 1 GB of drive space (although 10 GB is a recommended minimum if running
+  Jenkins as a <<docker,Docker>> container)
 
-* Java 8 (either JRE or JDK)
-* 256MB free memory
-* 1GB+ free disk space
+Recommended hardware configuration for a small team:
 
-Recommended Configuration for Small Team:
+* 1 GB+ of RAM
+* 50 GB+ of drive space
 
-* Java 8
-* 1GB+ free memory
-* 50GB+ free disk space
+Sofware requirements:
 
+* Java 8 - either a Java Runtime Environment (JRE) or a Java Development Kit
+  (JDK) is fine +
+  *Note:* This is not a requirement if running Jenkins as a <<docker,Docker>>
+  container.
 
-=== Experimentation, Staging, or Production?
 
-How you configure Jenkins will differ significantly depending on your intended use cases.
-This section is specifically targeted to initial use and experimentation.
-For other scenarios, see link:/doc/book/appendix/advanced-installation/[Advanced Jenkins Installation].
+== Installation platforms
 
-=== Stand-alone or Servlet?
+This section describes how to install/run Jenkins on different platforms and
+operating systems.
 
-Jenkins can run stand-alone in its own process using its own built-in web server (Jetty).
-It can also run as one servlet in an existing framework, such as Tomcat or Glassfish application servers.
-This section is specifically targeted to stand-alone install and execution.
-For other scenarios, see link:/doc/book/appendix/advanced-installation/[Advanced Jenkins Installation]
 
-== Installation
+=== Docker
+
+As described on the https://docs.docker.com/engine/docker-overview/[Docker
+overview] page, Docker is a platform for running applications in an isolated
```

Comment:
Try to avoid slow starts:
```
As described on the https://docs.docker.com/engine/docker-overview/[Docker
 +overview] page, Docker
```
change to:
```
https://docs.docker.com/engine/docker-overview/[Docker]
```

Reply:
Thanks for the tip - done!